Fix/499 bop summary

hoshmn opened the pull request:
todo:
- ~~CMSify~~


fixes #499

hoshmn commented on content/lang/federal.json:
```diff
@@ -1,30 +1,13 @@
-    }
+  "aggregates": {
+    "id": "aggregates",
+    "description": "The table below displays aggregate COVID-19 data for incarcerated people and staff in carceral facilities located in ${state}. The State DOC Data Grade represents our assessment of the transparency and quality of those data reported by ${state}'s state department of corrections.",
```
I realize some of this copy is state-specific, will fix when I migrate it to the CMS

hoshmn commented on src/templates/federal.js:
```diff
@@ -72,13 +73,24 @@ const useStyles = makeStyles((theme) => ({
+
+      // HACK: agg table overrides, need to restyle this page but this is here for now
+      "& #aggregates .MuiTableContainer-root": {
+        width: "calc(100vw - 6rem)",
```
@Lane see what you think of the styling, I figured we should let it expand to the full width available

ekmason commented:
@Lane  @hoshmn Small note - do we want to keep the "Federal Prisons" header in the summary table (screenshot below)? 
![image](https://user-images.githubusercontent.com/80919000/135266718-6814dbc5-68b3-4a81-bbbd-757ad967083a.png)

(This header wasn't present on Lane's original design shared in the ticket)
![image](https://user-images.githubusercontent.com/80919000/135266932-a54a785e-430e-4232-b8df-6b48a10e179d.png)

Lane commented:
@ekmason i think it's fine to keep the label on the table, it will provide some additional context if someone were to screenshot the table only.

@hoshmn looking good so far, a few changes:

- can we make the layout consistent with the state pages?  specifically:
  - spacing between header and top navigation
  - spacing between header text and intro paragraph
  - overall width of the content area (e.g. table spans full width of content area)
  - placement of the data grade

ideally these would match up:
![2021-09-29 10 28 47](https://user-images.githubusercontent.com/21034/135319429-eab39484-78d5-4381-b45b-9c9412444d76.gif)

also checking in, are the CMS pieces ready to test?

hoshmn commented:
thanks both for the feedback. @Lane i added styling fixes to match the states page.
>       * overall width of the content area (e.g. table spans full width of content area)
the federal page has a very restricted content area (45rem), hence all the css workarounds for elements that need to exceed that width. i currently have styles on the table to have it break out of the content area and grow to its full width when there's space.
> also checking in, are the CMS pieces ready to test?
yes! sorry, forgot to ping you when i added that

Lane commented:
thanks @hoshmn, i'm still seeing a layout shift in table width / placement of the section between the federal and state pages.

ideally, the federal + state templates should use similar content widths.  i've put in a refactor PR (#563) for the federal template that cleans up the layout and drops code that's no longer needed.  can you review and merge into this branch if it looks good to you?

once that's done, i'll merge this to development (along with other PR reviews) and do a similar refactor on the state template to remove the pieces we're no longer using

hoshmn commented:
Thanks for the refactor @Lane, looks great! Just merged it in.

> thanks @hoshmn, i'm still seeing a layout shift in table width / placement of the section between the federal and state pages.

The shift you're seeing should be addressed by the state template refactor, right? If not I'll fix.

Lane commented:
@hoshmn yep, the state refactor will bring the two layouts inline with each other.  thanks!
